### PR TITLE
fix: adjust emote region to have an owner with a buffer

### PR DESCRIPTION
**Pull request by lecoqjacob**

### CODE REVIEW

**Comment by lecoqjacob** [comment]
Peek 2023-06-22 10-40

File: assets/player/skins/fishy/fishy.player.yaml [hunk 1]
```diff
@@ -234,9 +234,6 @@ layers:
       emote_alarm:
         fps: *fps
         frames:
-          - 0
-          - 0
-          - 0
```
Comment (zicklag) [inline assets/player/skins/fishy/fishy.player.yaml:239]: we should remove these for the other players too.

**Review by zicklag — CHANGES_REQUESTED** [review]
Looks good, just have to remove the padding frames from the other player skins.

File: assets/player/skins/fishy/fishy.player.yaml (hunk 1 above)
Comment (lecoqjacob) [reply assets/player/skins/fishy/fishy.player.yaml:239]: Removed!

**Comment by lecoqjacob** [comment]
@zicklag removed the remaining buffers. Added the emote region to the bullet. We should probably extend the x value for the bullet emote region. It moves pretty fast and we want the players to emote a bit before it hits.

I was also noticing that we can only create rectangle regions for emote regions. Might look into how to create an anchor point so that the region can instead start at the very end of the bullet and not just surrounding it.

image

**Comment by zicklag** [comment]
Sounds like a good idea for the bullet region's offset. Do you want to try that before merging this, or should I just get this in and you can follow-up?

**Comment by lecoqjacob** [comment]
> Sounds like a good idea for the bullet region's offset. Do you want to try that before merging this, or should I just get this in and you can follow-up?

Merge this and ill create a ticket for follow up

**Review by zicklag — APPROVED** [review]

**Pull request merged**